internal/config/config_util_test.go

```diff
@@ -267,3 +267,7 @@ func Test_ListCacheTtlSecsToDuration_InvalidCall(t *testing.T) {
 	// Calling with invalid argument to trigger panic.
 	ListCacheTtlSecsToDuration(-3)
 }
+
+func Test_DefaultMaxParallelDownloads(t *testing.T) {
```

Comment:
I dont know how to add a test. But this is possibly checking for outliers like 0 or -1 which is also good. Is it possible to get the machine type ( only for the test ) and check if its as expected. ( like get machine name like n2-standard-32 and check 32 * 2 is the number. )

Reply:
I don't think that it's possible. We have to only rely on manual testing.